setup.cfg

```diff
@@ -72,13 +71,6 @@ docs =
   astroquery
   jplephem
 
-[options.package_data]
-sunpy = CITATION.rst
-sunpy.data = sunpyrc, test/*, test/*/*, data_manager/tests/db_testdata.csv
-sunpy.database = tests/test_table.txt
-sunpy.io = special/asdf/schemas/sunpy.org/sunpy/*/*.yaml, special/asdf/schemas/sunpy.org/sunpy/*/*/*.yaml
-sunpy.tests = figure_hashes_py36.json
-sunpy.net = vso/data/*, jsoc/data/*
 
```

Comment:
Why have these gone?

Reply:
Since we use setuptools_scm, it basically adds all the package data for us that is under the repo.